lib/winston/logger.js

```diff
@@ -42,29 +42,15 @@ class Logger extends Transform {
     this.configure(options);
   }
 
-  child(defaultRequestMetadata) {
+  child(overriddenRequestMetadata) {
     const logger = this;
     return Object.create(logger, {
-      write: {
-        value: function (info) {
-          const infoClone = Object.assign(
-            {},
-            defaultRequestMetadata,
-            info
-          );
-
-          // Object.assign doesn't copy inherited Error
-          // properties so we have to do that explicitly
-          //
-          // Remark (indexzero): we should remove this
-          // since the errors format will handle this case.
-          //
-          if (info instanceof Error) {
```

Comment:
Do we have a test where a child logger logs an `Error`?  Don't want to break that functionality if possible

Reply:
Not sure - Will verify.